src/components/KFileUpload/KFileUpload.vue

```diff
@@ -285,22 +286,38 @@ export default defineComponent({
 .k-file-upload {
   position: relative;
 
+  :deep(.k-input-label) {
+    pointer-events: none;
+
+    .label-tooltip span.kong-icon.kong-icon-help {
+      pointer-events: visible;
```

Comment:
If you keep this, instead of `visible` just set to `auto`

Reply:
Updated.